x-pack/test/api_integration/apis/management/index_management/templates.js

```diff
@@ -193,8 +193,8 @@ export default function ({ getService }) {
       });
 
       it('should parse the ES error and return the cause', async () => {
-        const templateName = `template-${getRandomString()}`;
-        const payload = getTemplatePayload(templateName, [getRandomString()]);
+        const templateName = `template-create-parse-es-error}`;
```

Comment:
I found these hardcoded strings very handy when I was debugging these tests by looking at the functional test runner logs.